cmd/kubeadm/app/master/manifests.go

```diff
@@ -96,10 +96,12 @@ func WriteStaticPodManifests(cfg *kubeadmapi.MasterConfiguration) error {
 			Env:           getProxyEnvVars(),
 		}, volumes...),
 		kubeScheduler: componentPod(api.Container{
-			Name:          kubeScheduler,
-			Image:         images.GetCoreImage(images.KubeSchedulerImage, cfg, kubeadmapi.GlobalEnvParams.HyperkubeImage),
-			Command:       getSchedulerCommand(cfg),
-			LivenessProbe: componentProbe(10251, "/healthz"),
+			Name:  kubeScheduler,
+			Image: images.GetCoreImage(images.KubeSchedulerImage, cfg, kubeadmapi.GlobalEnvParams.HyperkubeImage),
+			// TODO: Using non-standard port here so self-hosted scheduler can come up:
+			// Use the regular port if this is not going to be a self-hosted deployment.
+			Command:       getSchedulerCommand(cfg, 10260),
```

Comment:
Can we switch this depending on which deployment setup we're using instead of hardcoding like this?